app/routes/docs.admin.md

```diff
@@ -0,0 +1,11 @@
+# Administration
+
+The admin pages allow authorized users to manage users' Kafka group settings.
```

Comment:
Can we add a note here to explain that although this documentation is public because the source code itself is public that it is not important for normal users?

Reply:
Yea, thats a good call